### PR TITLE
changelog template, centralized docs repo

### PR DESCRIPTION
**Pull request by micbar**
Template needs more adjustment.

### CODE REVIEW

**Comment by micbar** [comment]
> In general, these steps will add a lot of extra complexity to the pipeline. Keep in mind that you will also have to maintain the theme versions over all extension repos.

What would you do instead?

File: .drone.star [hunk 1]
```diff
@@ -622,26 +622,78 @@ def website(ctx):
...
       {
-        'name': 'publish',
-        'image': 'plugins/gh-pages:1',
-        'pull': 'always',
+        'name': 'assets',
+        'image': 'byrnedo/alpine-curl',
+        'commands': [
+          'cd downstream',
+          'mkdir -p themes/hugo-geekdoc/',
+          'curl -L https://github.com/xoxys/hugo-geekdoc/releases/download/v0.1.7/hugo-geekdoc.tar.gz | tar -xz -C themes/hugo-geekdoc/ --strip-components=1'
```
Comment (micbar) [inline .drone.star]: Why do we need that step at all? Couldn't it be a static asset downstream?

Comment (xoxys) [reply .drone.star]: If you like so, sure, do it

**Comment by xoxys** [comment]
> What would you do instead?

If you want todo it within the extension pipelines there is not really another solution. As said, adding the commands to the Makefile will allow someone to "test" docs locally.

File: .drone.star [hunk 2]
```diff
@@ -622,26 +622,78 @@ def website(ctx):
...
+            ],
+          },
+        },
+      },
+      {
+        'name': 'copy-current-docs',
+        'image': 'webhippie/golang:1.13',
         'pull': 'always',
         'commands': [
-          'make docs',
```
[the remark this answers is not in the document]
Comment (micbar) [reply .drone.star]: 👍

File: .drone.star (hunk 1 above)
Comment (micbar) [reply .drone.star]: 👍

File: .drone.star [hunk 3]
```diff
@@ -622,26 +622,39 @@ def website(ctx):
...
     'steps': [
       {
-        'name': 'generate',
-        'image': 'webhippie/hugo:latest',
-        'pull': 'always',
+        'name': 'prepare',
+        'image': 'owncloud/alpine:latest',
         'commands': [
-          'make docs',
+          'apk add make',
```
Comment (tboerger) [inline .drone.star]: You should simply use a docker image that already includes make, why installing it on every pipeline run?

File: .drone.star [hunk 4]
```diff
@@ -622,26 +622,39 @@ def website(ctx):
...
+          'make docs-copy'
         ],
       },
       {
-        'name': 'publish',
-        'image': 'plugins/gh-pages:1',
-        'pull': 'always',
+        'name': 'test',
+        'image': 'klakegg/hugo:0.59.1-ext-alpine',
```
Comment (tboerger) [inline .drone.star]: Cool, now using some random guy docker image?

File: .drone.star [hunk 5]
```diff
@@ -622,26 +622,39 @@ def website(ctx):
...
+        'commands': [
+          'cd hugo',
+          'hugo-official',
+        ],
+      },
+      {
+        'name': 'list',
+        'image': 'iankoulski/tree',
```
Comment (tboerger) [inline .drone.star]: Cool, now using some random guy docker image?

File: .drone.star (hunk 3 above)
Comment (micbar) [reply .drone.star]: suggesttions?

File: .drone.star (hunk 4 above)
Comment (micbar) [reply .drone.star]: suggestions?

File: .drone.star (hunk 5 above)
Comment (micbar) [reply .drone.star]: better ideas?

File: .drone.star (hunk 3 above)
Comment (xoxys) [reply .drone.star]: I'll build one in owncloudci org....

File: .drone.star (hunk 4 above)
Comment (xoxys) [reply .drone.star]: Switched back to webhippie

File: .drone.star (hunk 5 above)
Comment (xoxys) [reply .drone.star]: I'll include tree in the custom one I'll prepare

Comment (tboerger) [reply .drone.star]: owncloud/ubuntu got tree installed, I think owncloud/alpine as well

**Comment by micbar** [comment]
@xoxys @refs So far, so good.

Squash and merge?

**Comment by xoxys** [comment]
@micbar LGTM

File: .drone.star [hunk 6]
```diff
@@ -622,26 +622,39 @@ def website(ctx):
     },
     'steps': [
       {
-        'name': 'generate',
+        'name': 'prepare',
+        'image': 'owncloudci/alpine:latest',
+        'commands': [
+          'apk add make',
```
Comment (refs) [inline .drone.star]: this is redundant, the image contains `make`:

```console
docker run -it owncloudci/alpine make
make: *** No targets specified and no makefile found.  Stop.
```

Comment (xoxys) [reply .drone.star]: You're right. Forgot to remove it.

Comment (xoxys) [reply .drone.star]: @refs Removed

**Pull request merged**